utility_network/configure-subnetwork-trace/README.md

```diff
@@ -0,0 +1,69 @@
+# Configure subnetwork trace
+
+Get a server-defined trace configuration for a given tier and modify its traversability scope, add new condition barriers and control what is included in the subnetwork trace result.
+
+![Configure Subnetwork Trace Sample Screenshot](ConfigureSubnetworkTrace.png)
+
+## Use case
+
+While some traces are built from an ad-hoc group of parameters, many are based on a variation of the trace configuration taken from the subnetwork definition. For example, an electrical trace will be based on the trace configuration of the subnetwork, but may add additional clauses to constrain the trace along a single phase. Similarly, a trace in a gas or electric design application may include features with a status of "In Design" that are normally excluded from trace results.
+
+## How to use the sample
+
+The sample loads with a server-defined trace configuration from a tier. Check or uncheck which options to include in the trace - such as containers or barriers. Use the selection boxes to define a new condition network attribute comparison. Click 'Trace' to run a subnetwork trace with this modified configuration from a default starting location.
```

Comment:
could still do with a reference to the "Add" button in this section

Reply:
Yep, this reference was added.